index.html

```diff
@@ -0,0 +1,880 @@
+          <div>
+            <button class="adds-btn">Table of Contents ↬</button>
+          </div>
+        </div>
+        <div class="grid-col-2">
+          <img
+            class="feature-panel-image-img"
+            src="https://res.cloudinary.com/indysigner/image/upload/v1542959380/shop/design-systems-hardcover-book.png"
+            loading="lazy"
+            alt="Design Systems, a new Smashing book"
+            width="256"
+            height="373"
+          />
+        </div>
+      </section>
+      <article class="more-posts">
+        <div class="post-text">
+          <h4>GUIDES</h4>
+          <h2>New Front-End Challenges To Master</h2>
+          <ul>
+            <li><a href="#"> CSS Grid Layout</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+            <li><a href="#"> Switching to HTTPS</a></li>
+            <li><a href="#"> HTTP/2 Guidelines</a></li>
+            <li><a href="#"> React & Vue.js</a></li>
+            <li><a href="#"> Redux For Designers</a></li>
+            <li><a href="#"> Vue Components</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+          </ul>
+        </div>
+        <div class="post-text">
+          <h4>GUIDES</h4>
+          <h2>New Front-End Challenges To Master</h2>
+          <ul>
+            <li><a href="#"> CSS Grid Layout</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+            <li><a href="#"> Switching to HTTPS</a></li>
+            <li><a href="#"> HTTP/2 Guidelines</a></li>
+            <li><a href="#"> React & Vue.js</a></li>
+            <li><a href="#"> Redux For Designers</a></li>
+            <li><a href="#"> Vue Components</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+          </ul>
+        </div>
+        <div class="post-text">
+          <h4>GUIDES</h4>
+          <h2>New Front-End Challenges To Master</h2>
+          <ul>
+            <li><a href="#"> CSS Grid Layout</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+            <li><a href="#"> Switching to HTTPS</a></li>
+            <li><a href="#"> HTTP/2 Guidelines</a></li>
+            <li><a href="#"> React & Vue.js</a></li>
+            <li><a href="#"> Redux For Designers</a></li>
+            <li><a href="#"> Vue Components</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+          </ul>
+        </div>
+        <div class="post-text">
+          <h4>GUIDES</h4>
+          <h2>New Front-End Challenges To Master</h2>
+          <ul>
+            <li><a href="#"> CSS Grid Layout</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+            <li><a href="#"> Switching to HTTPS</a></li>
+            <li><a href="#"> HTTP/2 Guidelines</a></li>
+            <li><a href="#"> React & Vue.js</a></li>
+            <li><a href="#"> Redux For Designers</a></li>
+            <li><a href="#"> Vue Components</a></li>
+            <li><a href="#"> Pattern Library</a></li>
+          </ul>
+        </div>
```

Comment:
You can keep the text in this section but change the bg color of the parent elements.